CAPT-1204 Created new screen - Playback email address from Teacher ID

martin-dfe opened the pull request:
[CAPT-1204](https://dfedigital.atlassian.net.mcas.ms/browse/CAPT-1204)

[CAPT-1204]: https://dfedigital.atlassian.net/browse/CAPT-1204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

slorek reviewed (changes requested):
We will need a new feature spec scenario for selecting the pre-filled email address as well.

martin-dfe committed:
Implemented the email selection screen

Added email validation

Added specs for new select-email screen

Mpved functionality from storing slugs into session to slug sequence

improved slug sequence
